neo/rawio/intanrawio.py

```diff
@@ -516,19 +661,35 @@ def read_rhd(filename):
     ordered_channels = []
 
     if version >= V("1.2"):
-        data_dtype = [("timestamp", "int32", BLOCK_SIZE)]
+        if file_format == "header-attached":
+            data_dtype = [("timestamp", "int32", BLOCK_SIZE)]
+        else:
+            data_dtype[6] = [("timestamp", "int32",)]
+            channel_number_dict[6] = 1
     else:
-        data_dtype = [("timestamp", "uint32", BLOCK_SIZE)]
+        if file_format == "header-attached":
+            data_dtype = [("timestamp", "uint32", BLOCK_SIZE)]
+        else:
+            data_dtype[6] = [("timestamp", "uint32",)]
+            channel_number_dict[6] = 1
```

Comment:
If we look here @h-mayorquin you can see that we add timestamp to our dtype for memmapping, but we don't add it to `ordered_channels`. So timestamps can't be accessible.